src/ILP/ilp-error.ts

```diff
@@ -1,5 +1,5 @@
-import { ServiceError, status } from 'grpc'
-import { Error as grpcWebError } from 'grpc-web'
+import { ServiceError } from 'grpc'
```

Comment:
Does importing ServiceError here still break the web system? We should avoid importing anything from GRPC in files where the browser will be consuming.

Reply:
per offline discussion - this doesn't break it since it's a TS construct and gets compiled out, but we decided to cut it since we're not really...using it here. also as mentioned, this avoids a `grpc` important